### ISSUE TITLE
Feature Request: Add Word Wrap Feature with Ability to Save Preference

### ISSUE THREAD

**Issue by wadallace** [warpdotdev/Warp#4094]
### Describe the solution you'd like?

I would like the ability to wrap output text to the next line via word wrap.  The word wrap should move the entire word to the next line instead of breaking the word.  I would like to be able to save this preference to automatically be enabled, but also would like the ability to toggle it with keyboard shortcuts. 

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Thanks for this feature request! 

To anyone else interested in this feature, please add a 👍 to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by SilentGlasses** [comment]
Yes, this is a real blocker for me as it's hard to read logs and other content that could potentially have long lines.

Also, it seems like currently lines break by character instead of words which makes things harder than not breaking at all.

![Warp_SoftWrapping_Broke](https://github.com/warpdotdev/Warp/assets/22822565/312f1011-3eb2-4697-ab17-35aff241e821)
